samples/security/spire/clusterspiffeid.yaml

```
@@ -0,0 +1,9 @@
+apiVersion: spire.spiffe.io/v1alpha1
+kind: ClusterSPIFFEID
+metadata:
+  name: example
+spec:
+  spiffeIDTemplate: "spiffe://{{ .TrustDomain }}/ns/{{ .PodMeta.Namespace }}/sa/{{ .PodSpec.ServiceAccountName }}"
+  podSelector:
+    matchLabels:
+      spiffe.io/spiffe-id: "true"
```

Nit, but to make what is happening here clearer, can the matchLabels have both of the labels on the sleep service?

```suggestion
      spiffe.io/spiffe-id: "true"
      app: sleep
```

---

Oh nevermind, you're overloading this for the gateway too.

In that case, can we change this to

```
spiffe.io/spire-workload-ca: "true"
```

---

Got it, thanks for explaining! Updated the label to `spiffe.io/spire-managed-identity: "true"`